integration/helm/ingress-nginx/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
Instead of packaging an nginx chart with each operator, can we provide a single chart that we can reference as a dependency? 

Host it within the [CoP Helm Charts](https://github.com/redhat-cop/helm-charts)?

Reply:
I am not sure what you mean here. Let's open a new issue for this. the ingress is not even necessary, but it was there in Trevor's setup.
@trevorbox  anything to add?